public/scripts/services/form.js

```diff
@@ -20,7 +20,7 @@
                     value = parseFloat(value);
                     break;
                 case 'string':
-                    value = value.toString();
+                    value = (typeof value === 'undefined' || value === null) ? "": value.toString();
```

Comment:
```suggestion
                    value = value?.toString() ?? '';
```

This might be a bit more elegant 🤔